lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -357,7 +357,7 @@ defmodule Kernel.SpecialForms do
       iex> Kernel.'+'(1, 2)
       3
 
-  Note that `Kernel."HELLO"` will be treated as a remote call and not an alias.
+  Note that `Kernel."function_name"` will be treated as a remote call and not an alias.
```

Comment:
We should change this to uppercase since the whole point is to show that `Kernel."Hello"` is not the same as `Kernel.Hello`.